src/ansys/geometry/core/_grpc/_services/v0/faces.py

```diff
@@ -504,3 +504,31 @@ def get_round_info(self, **kwargs) -> dict:  # noqa: D102
             "along_u": response.along_u,
             "radius": response.radius,
         }
+
+    @protect_grpc
+    def offset_faces(self, **kwargs) -> dict:  # noqa: D102
+        from ansys.api.dbu.v0.dbumodels_pb2 import EntityIdentifier
+        from ansys.api.geometry.v0.faces_pb2 import OffsetFacesRequest, OffsetFacesRequestData
+
+        # Create the request - assumes all inputs are valid and of the proper type
+        request = OffsetFacesRequest(
+            request_data=[
+                OffsetFacesRequestData(
+                    faces=[EntityIdentifier(id=face_id) for face_id in kwargs["face_ids"]],
```

Comment:
Use base_Grpc_id if possible -- we need to revisit all this _grpc folder to make sure we are using it everywhere, but in a separate, but in a separate PR